doc/source/running.rst

```diff
@@ -23,7 +23,7 @@ machines.
 The following uwsgi configuration file can be used::
 
   [uwsgi]
-  http = localhost:8041
```

Comment:
2.0.14, according to docs, http makes uwsgi act similar to nginx/apache? http://uwsgi-docs.readthedocs.io/en/latest/ThingsToKnow.html